API.md

```diff
@@ -101,23 +101,18 @@ This approach includes openmct using a simple script tag, resulting in a global
 variable named `openmct`. This `openmct` object is used subsequently to make 
 API calls. 
 
-Open MCT is packaged as a UMD (Universal Module Definition) module, so common 
```

Comment:
My bad, I think I told you to remove this, but I wasn't really thinking at the time 😄. Open MCT is still [built as a UMD module](https://github.com/nasa/openmct/blob/584c8b72b6e23aa8c85b79131f4da9b58d128402/webpack.config.js#L27).

Reply:
Done.